mern-ecommerce/controllers/order.js

```diff
@@ -61,3 +61,20 @@ exports.updateOrderStatus = (req, res) => {
     }
   );
 };
+
+exports.removeOrder = (req, res) => { 
+  Order.findByIdAndDelete(req.params.orderId) 
+    .then(deletedOrder => { 
+      if (!deletedOrder) {
+        return res.status(404).json({
+          error: "Order not found", 
+        });
+      }
+      res.json({ message: "Order deleted successfully" }); 
+    })
+    .catch(error => { 
+      res.status(500).json({
```

Comment:
In the `res.status(500).json({` line, it would be better to include a meaningful error message along with the error object. This will provide more information to the client about what went wrong. For example, `res.status(500).json({ error: errorHandler(error), message: 'Failed to delete order' });`.